nesta/packages/geo_utils/country_iso_code.py

```diff
@@ -25,8 +25,10 @@ def country_iso_code(country):
     country = str(country).title()
     for name_type in ['name', 'common_name', 'official_name']:
         query = {name_type: country}
-        try:
-            return pycountry.countries.get(**query)
+        try:            
+            result = pycountry.countries.get(**query)
+            if result is not None:
```

Comment:
This is super daft, but an edge case came into the new data just *today* as I was running this end-to-end in `dev` mode that doesn't really warrant a PR. The upshot is that `pycountry` returns `None` for this value, which for us is equivalent to saying "not found".